### PR TITLE
Correct framework usage in new docs

### PR DESCRIPTION
**Pull request by sergepilipchuk**

### CODE REVIEW

**Review by vladaskorohodova — APPROVED** [review]

File: concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/25 Customize Item Appearance.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+To customize the appearance of all Autocomplete suggestions, define an [item template](/api-reference/10%20UI%20Components/DataExpressionMixin/1%20Configuration/itemTemplate.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemTemplate'). 
 
-You can also enable [wrapItemText](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/wrapItemText.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#wrapItemText') if item text exceeds the drop-down list width to wrap the text.
+If your data is grouped, you may want to customize the group headings. Define a [group template](/api-reference/10%20UI%20Components/dxAutocomplete/1%20Configuration/groupTemplate.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupTemplate') for that purpose. 
 
----
-##### jQuery
+You can also enable [wrapItemText](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/wrapItemText.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#wrapItemText') if item text exceeds the drop-down list width to wrap the text.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/25 Customize Item Appearance.md]: You can also enable [wrapItemText](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/wrapItemText.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#wrapItemText') to wrap the text if the item's text exceeds the width of the drop-down list.

File: concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/25 Customize Item Appearance.md [hunk 2]
```diff
@@ -41,6 +47,12 @@ You can also enable [wrapItemText](/api-reference/10%20UI%20Components/dxDropDow
 
 ##### Vue
 
+To customize the appearance of all Autocomplete suggestions, define an [item template](/api-reference/10%20UI%20Components/DataExpressionMixin/1%20Configuration/itemTemplate.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemTemplate'). To customize an individual suggestion, use [custom templates](/concepts/05%20UI%20Components/zz%20Common/30%20Templates/10%20Custom%20Templates.md '/Documentation/Guide/UI_Components/Common/Templates/#Custom_Templates') property.
+
+If your data is grouped, you may want to customize the group headings. Define a [group template](/api-reference/10%20UI%20Components/dxAutocomplete/1%20Configuration/groupTemplate.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupTemplate') for that purpose.
+
+You can also enable [wrapItemText](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/wrapItemText.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#wrapItemText') if item text exceeds the drop-down list width to wrap the text.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/25 Customize Item Appearance.md]: Same here

File: concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/25 Customize Item Appearance.md [hunk 3]
```diff
@@ -63,6 +75,12 @@ You can also enable [wrapItemText](/api-reference/10%20UI%20Components/dxDropDow
...
+
+If your data is grouped, you may want to customize the group headings. Implement the [groupRender](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/groupRender.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupRender') or [groupComponent](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/groupComponent.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupComponent') function for that purpose.
+
+You can also enable [wrapItemText](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/wrapItemText.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#wrapItemText') if item text exceeds the drop-down list width to wrap the text.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/25 Customize Item Appearance.md]: and here

**Review by dirk-pieterse — APPROVED** [review]

File: concepts/Common/Security Considerations/20 HTML Encoding/20 Encode Template Data.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-Angular, Vue, and React always encode values interpolated in templates. With other frameworks and libraries, use a third-party sanitizing tool as follows:
+To encode values interpolated in templates, use a third-party sanitizing tool as follows:
```
Comment (dirk-pieterse) [inline concepts/Common/Security Considerations/20 HTML Encoding/20 Encode Template Data.md]: What do you mean by `interpolated`?

**Review by dirk-pieterse — APPROVED** [review]

File: concepts/Common/Security Considerations/20 HTML Encoding/20 Encode Template Data.md (hunk 4 above)
Comment (sergepilipchuk) [reply concepts/Common/Security Considerations/20 HTML Encoding/20 Encode Template Data.md]: I think this expression is misleading. So I have 
removed it

File: concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/25 Customize Item Appearance.md (hunk 1 above)
Comment (dirk-pieterse) [inline concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/25 Customize Item Appearance.md]:
```suggestion
You can also enable [wrapItemText](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/wrapItemText.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#wrapItemText') to wrap the text if the item's text exceeds the width of the drop-down list.
```

File: concepts/Common/Security Considerations/20 HTML Encoding/20 Encode Template Data.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-Angular, Vue, and React always encode values interpolated in templates. With other frameworks and libraries, use a third-party sanitizing tool as follows:
+To encode values in template, use a third-party sanitizing tool as follows:
```
Comment (dirk-pieterse) [inline concepts/Common/Security Considerations/20 HTML Encoding/20 Encode Template Data.md]:
```suggestion
To encode values in a template, use a third-party sanitizing tool as follows:
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**